DependencyInjection/JMSSerializerExtension.php

```diff
@@ -108,7 +108,7 @@ public function loadInternal(array $config, ContainerBuilder $container)
             $container->setParameter('jms_serializer.infer_types_from_doctrine_metadata', false);
         }
 
-        if (PHP_VERSION_ID >= 70400 && class_exists(TypedPropertiesDriver::class)) {
+        if (class_exists(TypedPropertiesDriver::class)) {
```

Comment:
why this change? we do not want to enable `TypedPropertiesDriver` for 7.2 or 7.4

Reply:
I wish I remembered. Will revert and see what happens